### ISSUE TITLE
where can i get Hans please?

### ISSUE THREAD

**Issue by strpstv** [nordseele/hans_2#1]
Hi! that seems like an awesome solution for me! i'd love to order one if possible? 
thank you!

**Comment by nordseele** [comment]
I'm sorry for the lack of communication. Unfortunately, I don't plan to commercialize or distribute a Hans 2 hardware, and it's highly unlikely that this project will be developed in the future. One reason is that I no longer use the ER-301, and secondly, I unfortunately don't have the time to provide support for users, write documentation, etc.

That being said, the code still works on a Raspberry Pi Pico W, and if you want to try it out, the easiest way is to create the MIDI IN circuit and the I2C output circuit going to the ER-301. These MIDI and I2C circuits are straightforward and in the public domain. When I have a moment, I will probably publish the schematic as it would be a good complement for using the code. However, I won't publish the files for the final PCB because, even though the prototype was functional, it wasn't ergonomically refined. For example, it didn't make sense to solder the RPI Pico directly to the PCB just to save 4mm in height.

In short, the PCBs from the prototype are not something I could distribute with a clear conscience in their current state; it wouldn't be fair to users. One never knows – if I ever feel like revisiting the project or if I find one of the prototypes that wasn't stripped of its components during the 2021 shortage...
